src/Angor/Client/Pages/Create.razor

```diff
@@ -587,11 +597,21 @@
         }
 
         var projects = storage.GetFounderProjects() ?? new List<FounderProject>();
+        projectCount = projects?.Count ?? 0;
+
         var keys = _walletStorage.GetFounderKeys();
 
-        if (projects.Count(p => !string.IsNullOrEmpty(p.CreationTransactionId)) >= keys.Keys.Count)
```

Comment:
Why is this exiting code not enough what are you trying to fix in this PR?

Reply:
if you go directly to the /create in the URL even if you have more than 15 projects it will allow it.
Same with /invest (if the time is over, there's still that hack).

Nevertheless, you are correct here, it's the same check (i'll remove the redundant check)

Reply:
ah I see yes this is probably needed ok